recipes-soletta/soletta/soletta_git.bb

```diff
@@ -105,6 +105,9 @@ do_configure_append() {
        sed -i -e 's/^ *FLOW_NODE_TYPE_GTK=.*/# FLOW_NODE_TYPE_GTK is not set/g' ${S}/.config
    fi
 
+   sed -i "s/^\(CONFIG_LDFLAGS=\).*/\1\"${TARGET_LDFLAGS}\"/g" ${S}/.config
```

Comment:
why not in `do_configure_prepend()` like others? these may impact on what internal configure detection will see (not the hash type, but eventually other ldflags)